package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "2025_1_quza",
+  "version": "1.0.0",
+  "description": "Репозиторий команды \"Quza\". Проект: Colab",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Давай добавим в эту секцию скрипт сборки, на фронте принято все скрипты вызывать черз npm. В данном случае вместо `./build.sh` будет `npm run build`
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "./build.sh"
  },
```

---

ок. добавил. везде далее, если без ответа коммент-просто в коде поправил